### PR TITLE
Deploying OSDs with cephadm

### PR DESCRIPTION
**Pull request by tbazant**
This update adds a new section with a description on how to deploy OSDs with cephadm.
Although it also includes more files, only the 'xml/deploy_cephadm.xml' is relevant.
Moreover, i imported and updated the Drivegroups content, hope it's still valid

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -908,6 +908,82 @@ ses-min4
...
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        The device does not contain a &bluestore; OSD.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        The device is larger than 5 GB.
```
Comment (lproven) [inline xml/deploy_cephadm.xml:947]:
```suggestion
        The device is larger than 5&nbsp;GB.
``

File: xml/deploy_cephadm.xml [hunk 2]
```diff
@@ -964,5 +1040,787 @@ ses-min4
...
+  <sect2 xml:id="drive-groups">
+   <title>&drvgrps;</title>
+   <para>
+    <emphasis>&drvgrps;</emphasis> specify the layouts of OSDs in the &ceph;
+    cluster. They are defined in a single YAML file. In this section, we will
+    use <filename>drive_groups.xml</filename> as an example.
+   </para>
+   <para>
+    An administrator should manually specify a group of OSDs that are
+    interrelated (hybrid OSDs that are deployed on solid state and spinners) or
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1053]: I think "spinners" is too informal and "solid state" is incomplete. I also note that the 'official' term used later seems to be `rotational`.

How about "hybrid OSDs that are deployed on a mixture of HDDs and SDDs"?

File: xml/deploy_cephadm.xml [hunk 3]
```diff
@@ -964,5 +1040,787 @@ ses-min4
...
+      <para>
+       By a disk vendor:
+      </para>
+<screen>
+vendor: <replaceable>DISK_VENDOR_STRING</replaceable>
+</screen>
+      <tip>
+       <title>Lowercase Vendor String</title>
+       <para>
+        Always lowercase the <replaceable>DISK_VENDOR_STRING</replaceable>.
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1185]: I do not like "lowercase" as a verb.

How about:

"Always enter the <replaceable>DISK_VENDOR_STRING</replaceable> in lower case."

File: xml/deploy_cephadm.xml [hunk 4]
```diff
@@ -964,5 +1040,787 @@ ses-min4
...
+       </para>
+      </tip>
+     </listitem>
+     <listitem>
+      <para>
+       Whether a disk is rotational or not. SSDs and NVME drives are not
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1191]:
```suggestion
       Whether a disk is rotational or not. SSDs and NVMe drives are not
```

File: xml/deploy_cephadm.xml [hunk 5]
```diff
@@ -964,5 +1040,787 @@ ses-min4
...
+     <para>
+      When using the ':' delimiter, you need to enclose the size in quotes,
+      otherwise the ':' sign will be interpreted as a new configuration hash.
+     </para>
+    </note>
+    <tip>
+     <title>Unit Shortcuts</title>
+     <para>
+      Instead of (G)igabytes, you can specify the sizes in (M)egabytes or
+      (T)erabytes as well.
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1268]: Does this mean marketing  mega/giga/tera or binary mebi/gibi/tibi?

File: xml/deploy_cephadm.xml [hunk 6]
```diff
@@ -964,5 +1040,787 @@ ses-min4
...
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        20 HDDs backed by 1 NVMe
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        2 HDDs backed by 1 SSD(db) and 1 NVMe(wal)
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1693]:
```suggestion
        2 HDDs backed by 1 SSD(db) and 1 NVMe (wal)
```

File: xml/admin_ceph_iscsi.xml [hunk 7]
```diff
@@ -20,7 +20,7 @@
  </info>
  <para>
   The chapter focuses on administration tasks related to the iSCSI Gateway. For
-  a procedure of deployment refer to <xref linkend="cha-ceph-as-iscsi"/>.
+  a procedure of deployment refer to DEAD_LINK <!-- <xref linkend="cha-ceph-as-iscsi"/> -->.
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/admin_ceph_iscsi.xml:23]: yes

File: xml/deploy_cephadm.xml (hunk 2 above)
Comment (tbazant) [inline xml/deploy_cephadm.xml:1054]:
```suggestion
    interrelated (hybrid OSDs that are deployed on a mixture of HDDs and SDDs) or
```

File: xml/deploy_cephadm.xml (hunk 3 above)
Comment (tbazant) [inline xml/deploy_cephadm.xml:1186]:
```suggestion
        Always enter the <replaceable>DISK_VENDOR_STRING</replaceable> in lower case.
```

File: xml/deploy_cephadm.xml (hunk 5 above)
Comment (tbazant) [reply xml/deploy_cephadm.xml:1269]: normal units, such as 1GB = 1024 * 1024 * 1024 bytes

File: xml/admin_ceph_cephfs.xml [hunk 8]
```diff
@@ -21,7 +21,7 @@
  <para>
   This chapter describes administration tasks that are normally performed after
   the cluster is set up and &cephfs; exported. If you need more information on
-  setting up &cephfs;, refer to <xref linkend="cha-ceph-as-cephfs"/>.
+  setting up &cephfs;, refer to DEAD_LINK <!-- <xref linkend="cha-ceph-as-cephfs"/> -->.
```
Comment (tbazant) [inline xml/admin_ceph_cephfs.xml:24]:
```suggestion
  setting up &cephfs;, refer to  <!-- DEAD_LINK <xref linkend="cha-ceph-as-cephfs"/> -->.
```

File: xml/admin_ceph_iscsi.xml (hunk 7 above)
Comment (tbazant) [inline xml/admin_ceph_iscsi.xml:23]:
```suggestion
  a procedure of deployment refer to <!-- DEAD_LINK <xref linkend="cha-ceph-as-iscsi"/> -->.
```

File: xml/admin_ceph_upgrade.xml [hunk 9]
```diff
@@ -74,7 +74,7 @@
      Your password must be changed to meet &productname; &productnumber;
      requirements. Ensure you change the username and password on
      <emphasis>all</emphasis> initiators as well. For more information on
-     changing your password, see <xref linkend="chap-auth-password"/>.
+     changing your password, see DEAD_LINK <!-- <xref linkend="chap-auth-password"/> -->.
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:77]:
```suggestion
     changing your password, see <!-- DEAD_LINK <xref linkend="chap-auth-password"/> -->.
```

File: xml/admin_gui_dashboard.xml [hunk 10]
```diff
@@ -1196,7 +1196,7 @@
     <title>More Information on &igw;s</title>
     <para>
      For more general information about &igw;s, refer to
-     <xref linkend="cha-ceph-as-iscsi"/> and <xref linkend="cha-ceph-iscsi"/>.
+     DEAD_LINK <!-- <xref linkend="cha-ceph-as-iscsi"/> and <xref linkend="cha-ceph-iscsi"/> -->.
```
Comment (tbazant) [inline xml/admin_gui_dashboard.xml:1199]:
```suggestion
     <!-- DEAD_LINK  <xref linkend="cha-ceph-as-iscsi"/> and <xref linkend="cha-ceph-iscsi"/> -->.
```

File: xml/admin_nfsganesha.xml [hunk 11]
```diff
@@ -51,7 +51,7 @@
   <title>Installation</title>
 
   <para>
-   For installation instructions, see <xref linkend="cha-as-ganesha"/>.
+   For installation instructions, see DEAD_LINK.
```
Comment (tbazant) [inline xml/admin_nfsganesha.xml:54]:
```suggestion
   For installation instructions, see <!-- DEAD_LINK -->.
```

File: xml/admin_saltcluster.xml [hunk 12]
```diff
@@ -994,7 +994,7 @@ Device Path Size rotates available Model name
     was upgraded by the update. If you want to eliminate the possibility of a
     forced reboot of potentially all nodes, either verify that the latest
     kernel is installed and running on &ceph; nodes, or disable automatic node
-    reboots as described in <xref linkend="ds-disable-reboots"/>.
+    reboots as described in DEAD_LINK.
```
Comment (tbazant) [inline xml/admin_saltcluster.xml:997]:
```suggestion
    reboots as described in <!-- DEAD_LINK -->.
```

File: xml/admin_ceph_cephfs.xml (hunk 8 above)
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/admin_ceph_cephfs.xml]: it's WIP & intentional, i will replace the strink later when i have a section to refer to

**Pull request merged**